### PR TITLE
feat(repl): Make SvelteKit agnostic

### PR DESCRIPTION
**Pull request by PuruVJ**
Should not depend on SvelteKit or Vite anymore

svelte-package is updated

No breaking changes

### CODE REVIEW

File: packages/repl/package.json [hunk 1]
```diff
@@ -5,38 +5,60 @@
...
+	},
+	"exports": {
+		"./package.json": "./package.json",
+		".": {
+			"types": "./dist/index.d.ts",
+			"svelte": "./dist/index.js",
+			"default": "./dist/index.js"
+		}
+	},
+	"typesVersions": {
```
Comment (benmccann) [inline packages/repl/package.json:54]: might not need this?

File: packages/repl/src/lib/workers/bundler/index.js [hunk 2]
```diff
@@ -339,8 +340,7 @@ async function get_bundle(uid, mode, cache, lookup) {
 }
 
 async function bundle({ uid, components }) {
-	if (import.meta.env.PROD) {
-		console.clear();
+	if (BROWSER) {
```
Comment (benmccann) [inline packages/repl/src/lib/workers/bundler/index.js:343]: should this be `!DEV`? I haven't looked at what it's actually doing, but that would match what it was before

File: packages/repl/svelte.config.js [hunk 3]
```diff
@@ -5,7 +5,4 @@ export default {
 	kit: {
 		adapter: adapter()
 	},
-	package: {
-		exports: (file) => file === 'index.js'
-	}
-};
+	};
```
Comment (benmccann) [inline packages/repl/svelte.config.js:8]: not sure if the semicolon is required? can definitely remove a couple spaces though
```suggestion
};
```

File: packages/repl/package.json (hunk 1 above)
Comment (PuruVJ) [reply packages/repl/package.json:54]: For people using with TS 4, still might need it

File: packages/repl/src/lib/workers/bundler/index.js (hunk 2 above)
Comment (PuruVJ) [reply packages/repl/src/lib/workers/bundler/index.js:343]: Oops. Mistake 😅

File: pnpm-lock.yaml [hunk 4]
```diff
@@ -4273,7 +4692,7 @@ packages:
       - rollup
     dev: true
 
-  /vite/4.1.1:
+  /vite@4.1.1:
```
Comment (benmccann) [inline pnpm-lock.yaml:4695]: would be nice to use 4.2.1 everywhere so that we don't have multiple versions

File: pnpm-lock.yaml [hunk 5]
```diff
@@ -4146,72 +4553,78 @@ packages:
...
 
-  /typedarray-to-buffer/3.1.5:
+  /typedarray-to-buffer@3.1.5:
     resolution: {integrity: sha512-zdu8XMNEDepKKR+XYOXAVPtWui0ly0NtohUscw+UmaHiAWT8hrV1rr//H6V+0DvJ3OQ19S979M0laLfX8rm82Q==}
     dependencies:
       is-typedarray: 1.0.0
     dev: false
 
-  /typescript/4.9.5:
+  /typescript@4.9.5:
```
Comment (benmccann) [inline pnpm-lock.yaml:4611]: would be nice to use 5.0.4 everywhere so that we don't have multiple versions

File: pnpm-lock.yaml [hunk 6]
```diff
@@ -4045,19 +4433,40 @@ packages:
...
+      svelte: ^3.55
+      typescript: ^4.9.4 || ^5.0.0
+    dependencies:
+      dedent-js: 1.0.1
+      pascal-case: 3.1.2
+      svelte: 3.58.0
+      typescript: 5.0.4
+    dev: true
+
+  /svelte@3.55.1:
```
Comment (benmccann) [inline pnpm-lock.yaml:4460]: looks like we've got multipe versions of svelte as well

File: pnpm-lock.yaml (hunk 4 above)
Comment (PuruVJ) [reply pnpm-lock.yaml:4695]: Other one is site-kit, which is better to not touch IMO. Let's leave it like this for a while IG? Then if we move site-kit back here, it should not be an issue

File: pnpm-lock.yaml (hunk 6 above)
Comment (PuruVJ) [reply pnpm-lock.yaml:4460]: Hmm will try updating some deps

**Review by benmccann — APPROVED** [review]

**Comment by PuruVJ** [comment]
Ok where is Changeset bot 🤔

**Comment by PuruVJ** [comment]
Merging, as it seems Changeset bot doesn't like commenting in this repo

**Pull request merged**